balboa-agent/src/main/java/com/socrata/balboa/agent/HttpOrMq.java

```diff
@@ -0,0 +1,19 @@
+package com.socrata.balboa.agent;
+
+public enum HttpOrMq {
```

Comment:
It's a Java enum because plain Scala enums are really annoying to work with. If you'd prefer, I can write it as a sealed trait.